system/lambdaLoadTest/playlist.xml

```diff
@@ -136,7 +136,7 @@
 			<impl>openj9</impl>
 			<impl>ibm</impl>
 		</impls>
-		<platformRequirements>bits.64,^arch.arm</platformRequirements>
+		<platformRequirements>bits.64,^arch.arm,^spec.linux_ppc-64_le</platformRequirements>
```

Comment:
Is spec.linux_ppc-64_le only going to exclude XL and not compressed refs?
Also it needs to be excluded for xlinux XL as well.

Reply:
> Is spec.linux_ppc-64_le only going to exclude XL and not compressed refs?

Yes : https://github.com/AdoptOpenJDK/openjdk-tests/blob/28a458bbc492b8d287f78097d4d6ba041e4b7787/buildenv/jenkins/openjdk_tests#L45

Reply:
Excluded xLinux XL as well. Please refresh PR to check. 

Test: /Test_grinder/job/Grinder/5990/console